src/java/org/jivesoftware/openfire/archive/ConversationUtils.java

```diff
@@ -352,7 +352,7 @@ private String formatJID(boolean html, JID jid) {
         if (html) {
             UserManager userManager = UserManager.getInstance();
             if (XMPPServer.getInstance().isLocal(jid) &&
-                userManager.isRegisteredUser(jid.getNode())) {
+                userManager.isRegisteredUser(jid, true)) {
```

Comment:
The old code checks if the JID belongs to a user of the _local_ domain. The replacement code explicitly tries to check other domains. I believe that the second argument value should be `false`.

Reply:
Yep, absolutely inverted logic.

Reply:
Fixed